cleanlab_studio/internal/api/api.py

```diff
@@ -636,6 +636,43 @@ def list_all_enrichment_projects(api_key: str) -> List[JSONDict]:
     return all_projects
 
 
+def enrichment_preview(
+    api_key: str,
+    new_column_name: str,
+    project_id: str,
+    prompt: str,
+    constrain_outputs: Optional[List[str]] = None,
+    extraction_pattern: Optional[str] = None,
+    indices: Optional[List[int]] = None,
+    optimize_prompt: Optional[bool] = None,
+    quality_preset: Optional[TLMQualityPreset] = None,
+    replacements: Optional[List[Dict[str, str]]] = None,
```

Comment:
I think we did a good job to minimizing any "weird" type to the base layer. 
The reason I didn't put TLMOptions for `tlm_options` is the same reason. low-level api should have basic type so that it can adapt all the high-level function needs.